src/frontend/src/lib/components/hero/HeroSignIn.svelte

```diff
@@ -2,9 +2,20 @@
 	import { i18n } from '$lib/stores/i18n.store';
 	import { signIn } from '$lib/services/auth.services';
 	import ButtonIC from '$lib/components/ui/ButtonIC.svelte';
+	import IconHeaderICP from '$lib/components/icons/IconHeaderICP.svelte';
+	import IconHeaderBTC from '$lib/components/icons/IconHeaderBTC.svelte';
+	import IconHeaderETH from '$lib/components/icons/IconHeaderETH.svelte';
+	import IconHeaderUSDC from '$lib/components/icons/IconHeaderUSDC.svelte';
 </script>
 
-<div class="mt-6 xl:mt-12 mb-10 pt-2">
+<div class="flex -space-x-2.5 mt-6 xl:mt-12">
+	<div class="z-0"><IconHeaderICP /></div>
+	<div class="z-10"><IconHeaderBTC /></div>
+	<div class="z-20"><IconHeaderETH /></div>
+	<div class="z-30"><IconHeaderUSDC /></div>
```

Comment:
Just to be sure, there is no z-index conflicts with the backdrop of the busy spinner that is displayed when user sign-in right?

Reply:
you mean this one, right?

<img width="881" alt="Screenshot 2024-07-22 at 12 52 53" src="https://github.com/user-attachments/assets/7864e090-ccad-42e5-bcc6-61cc4c5587a3">

Reply:
Yes. Good!

Never too sure with z-indexes 😅

Reply:
you were right! it gave issues in the end! see PR #1818